cmd/litcli/ln.go

```diff
@@ -237,74 +236,80 @@ type resultStreamWrapper struct {
 //
 // NOTE: This method is part of the PaymentResultStream interface.
 func (w *resultStreamWrapper) Recv() (*lnrpc.Payment, error) {
-	resp, err := w.stream.Recv()
-	if err != nil {
-		return nil, err
-	}
-
-	res := resp.Result
-	switch r := res.(type) {
-	// The very first response might be an accepted sell order, which we
-	// just print out.
-	case *tchrpc.SendPaymentResponse_AcceptedSellOrder:
-		quote := r.AcceptedSellOrder
+	// printQuote unmarshals and prints an accepted quote.
+	printQuote := func(quote *rfqrpc.PeerAcceptedSellQuote) error {
 		rpcRate := quote.BidAssetRate
 		rate, err := rpcutils.UnmarshalRfqFixedPoint(rpcRate)
 		if err != nil {
-			return nil, fmt.Errorf("unable to unmarshal fixed "+
-				"point: %w", err)
+			return fmt.Errorf("unable to unmarshal fixed point: %w",
+				err)
 		}
 
 		amountMsat := lnwire.MilliSatoshi(w.amountMsat)
 		milliSatsFP := rfqmath.MilliSatoshiToUnits(amountMsat, *rate)
 		numUnits := milliSatsFP.ScaleTo(0).ToUint64()
 
-		// If the calculated number of units is 0 then the asset rate
-		// was not sufficient to represent the value of this payment.
+		// The purpose of this function is just to print, so let's avoid
```

Comment:
This should not be a comment in the code (since it references deleted code), it's more like a remark on the PR or a commit description.